RULES.md

```diff
@@ -34,6 +34,7 @@ Rules are declared in the [`Notice` module](https://github.com/MobilityData/gtfs
 | [E030](#E030) | Inconsistent field `agency_timezone` | 
 | [E031](#E031) | Invalid `agency_id` | 
 | [E032](#E032) | `calendar.txt` `end_date` is before `start_date` |
+| [E032](#E033) | Reference to non-existing `route_id` in `trips.txt` |
```

Comment:
I would suggest we adopt a broader definition of this rule, like "`route_id` missing from routes.txt" or "Invalid route_id" or "route_id does not exist in routes.txt" that would cover all instances of a route_id used in a different file that doesn't exist in routes.txt.

route_id is used in many several files, and given the number of rules that will reference different IDs across different files, we're doing to end up with a lot of rules just of bad ID references. I think this unnecessarily clutters the rule list and makes it harder to find other semantic rules. Plus, we can trace which file the error is coming from from the message itself, right?

Also, related to the terminology of the rule name, we ended up using the following convention in the RT validator:
"GTFS-rt trip_id does not exist in GTFS data"

After a lot of discussion this seemed easier to understand that using words like "reference" or "non-existing" (which we also used at some point).

Reply:
True, and yes in the notice description we can track back the problematic file.

Reply:
I agree with the stated points, I suppose they are not implemented yet and will pause my review of this PR